pkgs/development/interpreters/php/default.nix

```diff
@@ -19,7 +19,8 @@ let
 
       enableParallelBuilding = true;
 
-      buildInputs = [ flex bison pkgconfig systemd ];
+      buildInputs = [ flex bison pkgconfig ]
+      ++ lib.optional stdenv.isLinux [systemd ];
 
```

Comment:
phpfpm now uses the ```sd_notify``` interface to signal its readiness during startup. Added in #19264. Apologies for breaking the build for mac users!